Makefile

```
@@ -0,0 +1,16 @@
+SHELL       := /bin/sh
+IMAGE       := keppel.eu-de-1.cloud.sap/ccloud/alertmagnet
+VERSION     := 0.1
```

```suggestion
SHELL       ?= /bin/sh
IMAGE       ?= keppel.eu-de-1.cloud.sap/ccloud/alertmagnet
VERSION     ?= 0.1
```
otherwise you cannot overwrite the variables from make